src/scripts/main.js

```diff
@@ -1,3 +1,30 @@
 'use strict';
 
+const array = [];
```

Comment:
Consider giving the array a more descriptive name, such as 'populationNumbers', to clarify its purpose.